tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -329,7 +330,12 @@ const waitForLoaders = async () => {
 };
 
 const waitForAngularLoaded = async (timeout = 40000) => {
-  await (await hamburgerMenu()).waitForDisplayed({ timeout });
+  const windowSize = await browser.getWindowSize();
+  if (windowSize.width <= 500 && windowSize.height <= 700) {
+    await (await toolBar()).waitForDisplayed({ timeout });
+  } else {
+    await (await hamburgerMenu()).waitForDisplayed({ timeout });
+  }
```

Comment:
I think that it is a better idea to modify the test than to change this method. 

The problem with the test is [this section](https://github.com/medic/cht-core/blob/160-e2e-tests-refactor-top-bar/tests/e2e/default-mobile/navigation/more-options-menu.wdio-spec.js#L65):
```.js
await commonPage.goToPeople(patient._id);
```

We should first go to people's tab and then search for the patient when we are on mobile, like this:
```.js
await commonPage.goToPeople();
await contactPage.selectLHSRowByText(patient.name);
```

And since we are using the search we should use the `patient.name` and no `patient._id`

With these changes you will need to modify other sections of the test, we will need to add a specific name for patient and contact, [here](https://github.com/medic/cht-core/blob/160-e2e-tests-refactor-top-bar/tests/e2e/default-mobile/navigation/more-options-menu.wdio-spec.js#L17-L21) and [here](https://github.com/medic/cht-core/blob/160-e2e-tests-refactor-top-bar/tests/e2e/default-mobile/navigation/more-options-menu.wdio-spec.js#L29-L31), because right now both have the same name, it would look like this.

```.js
const contact = personFactory.build({
    name: 'contact_name',
    phone: '+12068881234',
    place: health_center._id,
    parent: { _id: health_center._id, parent: health_center.parent },
  });

const patient = personFactory.build({
    name: 'patient_name',
    parent: health_center
  });
```

And then you will need to add a beforeEach to go to the base, for example, because the search needs to be cleared for the second test, it can be something like this:
```.js
beforeEach(async () => await commonPage.goToBase());
```